configuring.tex

```diff
@@ -3,6 +3,11 @@ \chapter{Configuring your MEGA65}
 
 \section{Preparing for First Use: Formatting SD Cards}
 
+\textbf{NOTE}: For your convenience, your MEGA65 comes with an SD card with all essential 
+files placed on it. So you may prefer to skip this section and jump to the on-boarding section 
```

Comment:
comma here between it and So?

Reply:
Hmm, I'm 50/50 on it, but ok, let's try it that way :)